clients/dotnet-core-rest/src/Tika.RestClient/Features/Acls/AclsClient.cs

```diff
@@ -12,24 +13,26 @@ public class AclsClient : IAclsClient
     {
         private const string ACLS_ROUTE = "/access-control-lists";
         private readonly HttpClient _httpClient;
+        private readonly ClientOptions _clientOptions;
 
-        public AclsClient(HttpClient httpClient)
+        public AclsClient(HttpClient httpClient, ClientOptions options)
         {
             _httpClient = httpClient;
+            _clientOptions = options;
         }
         
-        public async Task<IEnumerable<Acl>> GetAllAsync()
+        public async Task<IEnumerable<Acl>> GetAllAsync(string clusterId = null)
         {
             var httpResponseMessage = await _httpClient.GetAsync(
-                new Uri(ACLS_ROUTE, UriKind.Relative)
+                new Uri(Utilities.MakeUrl(_clientOptions, ACLS_ROUTE, clusterId), UriKind.Absolute)
```

Comment:
Maybe call this a helper? and not a utility since its a very specific function that solves a very specific problem?